providers/tailscale/resources/tailscale.lr

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+option provider = "go.mondoo.com/cnquery/v11/providers/tailscale"
+option go_package = "go.mondoo.com/cnquery/v11/providers/tailscale/resources"
+
+// The Tailscale provider
+tailscale {
+  // List devices in a tailnet
```

Comment:
I think we should expose the organization identifier as well as the tailnet name here